pkg/export/attributes/attr_defs.go

```diff
@@ -137,6 +137,7 @@ func getDefinitions(groups AttrGroups) map[Section]AttrReportGroup {
 			attr.K8sPodStartTime:    true,
 			attr.K8sClusterName:     true,
 			attr.K8sOwnerName:       true,
+			attr.K8sKind:            true,
```

Comment:
Is this equivalent to the concept of `k8s.src.type`/`k8s.dst.type` of network metrics?

In that case, I would maybe add equivalent `k8s.{src,dst}.kind`  and `k8s.{src,dst}.owner.kind` metrics in network metrics, and mark the `-.type` suffixed metrics as deprecated in Beyla 3.0.

Of course this is not for this PR but we should add an issue to keep track of it, just for naming consistency.

Reply:
Yeah it's the same. I used the name kind to match the service filter we have owner_kind.

Reply:
We can clean up in a follow-up PR, but I think it might be good to do so once we have network flows in OTel spec?